WordPress/src/main/java/org/wordpress/android/viewmodel/pages/PageListViewModel.kt

```diff
@@ -401,14 +401,20 @@ class PageListViewModel @Inject constructor(
                 pagesViewModel.site,
                 pageModel.remoteId
         )
-        return ItemUiStateData(labels, labelColor, progressBarUiState, showOverlay, actions)
+        val subtitle = when(pageModel.remoteId) {
+            pagesViewModel.site.pageOnFront -> R.string.site_settings_homepage
+            pagesViewModel.site.pageForPosts -> R.string.site_settings_posts_page
+            else -> null
+        }
+        return ItemUiStateData(labels, labelColor, progressBarUiState, showOverlay, actions, subtitle)
     }
 
     private data class ItemUiStateData(
         val labels: List<UiString>,
         @ColorRes val labelsColor: Int?,
         val progressBarUiState: ProgressBarUiState,
         val showOverlay: Boolean,
-        val actions: Set<Action>
+        val actions: Set<Action>,
+            val subtitle: Int? = null
```

Comment:
nit: extra spaces here

Reply:
thanks, I've fixed that and one other lint error